package.json

```diff
@@ -9,11 +9,13 @@
     "prebuild": "rimraf dist",
     "build": "nest build",
     "format": "prettier --write \"src/**/*.ts\" \"test/**/*.ts\"",
+    "format:check": "prettier --check \"src/**/*.ts\" \"test/**/*.ts\"",
```

Comment:
Perhaps we could make the `check` / `fix` naming consistent across the `format` and `lint` jobs? So:

- `format:check` and `format:fix`
- `lint:check` and `lint:fix`

Reply:
Good idea, I've added that in [55f1030](https://github.com/ably-labs/test-observability/pull/16/commits/55f10304538058d44cdfc75953a0d757cbe8d83e)